capellambse/diagram/capstyle.py

```diff
@@ -174,7 +174,6 @@ class in the form::
     "_CAP_Activity_Orange": RGB(247, 218, 116),
     "_CAP_Activity_Orange_min": RGB(255, 255, 197),
     "_CAP_Actor_Blue": RGB(198, 230, 255),
-    "_CAP_Actor_Blue_label": RGB(0, 0, 0),
```

Comment:
This map contains all the color names known to Capella, so that the style definitions below can use the exact names used in Capella's stylesheet.

This does mean that there are dups, but I guess it is what it is.